apps/web/src/server/lib/msGraph.js

```diff
@@ -0,0 +1,42 @@
+import { createHttpLoggerHooks } from '@pins/platform';
+import config from '@pins/applications.web/environment/config.js';
+import got from 'got';
```

Comment:
this file is basically a copy of the file "request.js", but with a different prefixUrl

Reply:
can you rename the file ms-graph for consistency

Reply:
renamed msgraphRequest